devito/core/autotuning.py

```diff
@@ -24,6 +29,7 @@ def autotune(operator, arguments, tunable, mode='basic'):
 
     # Squeeze dimensions to minimize auto-tuning time
     iterations = FindNodes(Iteration).visit(operator.body)
+    dim_mapper = {i.dim.name: i.dim for i in iterations}
```

Comment:
I think the right thing to do is to change the test so that we intercept ``autotune=True`` and we call the autotuner explicitly from there. In fact, the test in ``test_dle.py`` is supposed to test the autotuning feature, not if it's called...
Then from ``test_operator.py`` we could test *if* the autotuner gets called -- but how ? Anyway, not part of this PR, but something to keep in mind (suggestions welcome)